lib/phoenix/live_dashboard/page_builder.ex

```diff
@@ -762,6 +762,58 @@ defmodule Phoenix.LiveDashboard.PageBuilder do
     """
   end
 
+  @doc """
+  List of label value.
+
+  You can see it in use in the modal in Ports or Processes page.
+  """
+  slot :elem, required: true, doc: "Value for each element of the list" do
+    attr :label, :string, required: true, doc: "Label for the elem"
+  end
+
+  def label_value_list(assigns) do
```

Comment:
I think this component deserves a better name 🤣

Reply:
Current name looks alright to me.